sound/soc/intel/boards/tgl_rt1308.c

```diff
@@ -0,0 +1,331 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Xiuli Pan <xiuli.pan@linux.intel.com>
```

Comment:
Authot info in one place is good enough. There's already a MODULE_AUTHOR() at the end

Reply:
Yes, I never understood this desire from developers to have their name in a file. there is already GIT as a tamper-proof record of contributions, and the MODULE_AUTHOR is another way of bragging. If it was just me I'd remove this.